tracee-ebpf/main.go

```diff
@@ -26,6 +29,7 @@ var traceeInstallPath string
 var buildPolicy string
 
 // These vars are supposed to be injected at build time
+//go:embed dist/tracee.bpf.tar.gz
 var bpfBundleInjected string
```

Comment:
I don't think string is the best type here. If we want to keep the legacy workflow, we should change to `[]byte`, but I perhaps it'll be better to embed as `FS` then we can eliminate that entire workflow from the makefile and also the unpackBPFBundle code. WDYT?

Reply:
It's an interesting point, I need to investigate it further.

Reply:
Thanks for the idea – I implemented it here: https://github.com/aquasecurity/tracee/pull/596/commits/abb0987bd91d35eb73a09f26c3ecf3718e332318 this trimmed some fat.